dashboard/app/controllers/learning_goal_evaluations_controller.rb

```diff
@@ -0,0 +1,33 @@
+class LearningGoalEvaluationsController < ApplicationController
+  def create
+    learning_goal_evaluation = LearningGoalEvaluation.new(learning_goal_evaluation_params)
+
+    learning_goal_evaluation.save!
+
+    render json: learning_goal_evaluation
+  end
+
+  def update
+    params.require(:id)
+    learning_goal_evaluation = LearningGoalEvaluation.find_by_id(params[:id])
+    if learning_goal_evaluation
+      learning_goal_evaluation.update(learning_goal_evaluation_params)
+      render json: learning_goal_evaluation
+    end
+  end
+
+  private def learning_goal_evaluation_params
+    lgep = params.transform_keys(&:underscore)
+    lgep = lgep.permit(
+      :user_id,
+      :teacher_id,
```

Comment:
`teacher_id` is the id of the teacher who wrote the feedback so I think we should set this to be `current_user.id` and reject the requests if the user isn't a teacher.

On that note, we'll probably want to add authentication to these routes, likely using `ability.rb` and adding `authorize_resource` at the top of this file. (Search for "TeacherFeedback" in [ability.rb](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/ability.rb) for examples, though it's not exactly the same.) We could do this as a follow up if we make the restrictions I suggested above but it may not actually save any work to do so